ludwig/distributed/ddp.py

```diff
@@ -77,7 +77,9 @@ def gather_all_tensors_fn(cls) -> Optional[Callable]:
 
     @classmethod
     def get_ray_trainer_backend(cls, **kwargs) -> Optional[Any]:
-        return "torch"
+        from ray.train.torch import TorchConfig
+
+        return TorchConfig()
```

Comment:
@tgaddair Just updated - I think this should be fine for now. `TorchConfig` does have some custom attributes but I think we can skip overriding them from now. Eventually, we can add this to the config.